test/unit/rules/no-down-event-binding-test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const { ERROR_MESSAGE } = require('../../../lib/rules/no-down-event-binding');
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-down-event-binding',
+
+  config: true,
+
+  good: [
+    "<div {{on 'keyup' this.doSomething}}></div>",
+    "<div {{action this.doSomething on='keyup'}}></div>",
+  ],
+
+  bad: [
```

Comment:
What about the `onkeydown` DOM event?

https://www.w3schools.com/jsref/event_onkeydown.asp

Reply:
Yeah, that would be caught by this -- the tests aren't necessarily exhaustive. Would you prefer I add test cases for that event specifically? This rule disallows any `__down` event.

Reply:
I don't see any test case with pure HTML. Can you add this?

```html
<input type="text" onkeydown="myFunction()">
```

Reply:
Oh! Great point. I'll add a test for that!